src/allmydata/test/web/test_grid.py

```diff
@@ -47,40 +58,46 @@ class Grid(GridTestMixin, WebErrorMixin, ShouldFailMixin, testutil.ReallyEqualMi
     def CHECK(self, ign, which, args, clientnum=0):
         fileurl = self.fileurls[which]
         url = fileurl + "?" + args
-        return self.GET(url, method="POST", clientnum=clientnum)
+        return self.GET(url, method="POST", clientnum=clientnum).addCallback(str, "utf-8")
```

Comment:
Over on https://github.com/tahoe-lafs/tahoe-lafs/pull/890 I found myself doing a lot of converting inputs to CHECK to be bytestrings. Is this what avoids that here?

Reply:
Yeah.